polytracker/custom_abi/dfsan_abilist.txt

```diff
@@ -154,7 +154,8 @@ fun:llrintl=functional
 
 # Functions that produce an output that does not depend on the input (shadow is
 # zeroed automatically).
-fun:__assert_fail=discard
+fun:__assert_fail=uninstrumented
```

Comment:
What does "uninstrumented" result in? Does this mean the function is stubbed and polytracker does not interact with it?

Reply:
>The pass treats every function in the uninstrumented category in the ABI list file as conforming to the native ABI. Unless the ABI list contains additional categories for those functions, a call to one of those functions will produce a warning message, as the labelling behavior of the function is unknown.

Source: https://clang.llvm.org/docs/DataFlowSanitizer.html#abi-list

Does this answer your question?

Reply:
Thank you! I didn't realize this is/was the same as the regular DFSan ABI list, my bad.